app/components/works/contributor_row_component.rb

```diff
@@ -66,11 +66,19 @@ def model
       form.object.class.method_defined?(:model) ? form.object.model : form.object
     end
 
+    def contributor_remove_label
+      "Remove #{contributor_name.empty? ? "blank contributor" : contributor_name}"
+    end
+
+    def contributor_name
+      "#{form.object.first_name} #{form.object.last_name}".strip
+    end
+
```

Comment:
this may be different for organization vs author contributors, and so this logic may need to be a bit more invovled